Bring the docs to the style guide. (#29)
iotaledger/inx-participation: documentation/docs/welcome.md

# Welcome to INX-Participation

> There is no dedicated structure for a voting event on the Tangle. Instead, it is represented by multiple individual transactions with certain payloads. To access the state of a voting event and the amount and distribution of the votes, you would have to find all these transactions in the Tangle and count each vote. INX-Participation does that you: it builds a database of all voting transactions for given events and provides REST API endpoints for clients to access this information. Node operators manually select which events they wish to track.

Reviewer: ```suggestion
There is no dedicated structure for a voting event on the Tangle. Instead, it is represented by multiple individual transactions with specific payloads. To access the state of a voting event and the amount and distribution of the votes, you would have to find all these transactions in the Tangle and count each vote. INX-Participation does that for you: it builds a database of all voting transactions for given events and provides REST API endpoints for clients to access this information. Node operators manually select which events they wish to track.
```

> There is no dedicated structure for a voting event on the Tangle. Instead, it is represented by multiple individual transactions with certain payloads. To access the state of a voting event and the amount and distribution of the votes, you would have to find all these transactions in the Tangle and count each vote. INX-Participation does that you: it builds a database of all voting transactions for given events and provides REST API endpoints for clients to access this information. Node operators manually select which events they wish to track.

Reviewer: ```suggestion
There is no dedicated structure for a voting event on the Tangle. Instead, it is represented by multiple individual transactions with specific payloads. To access the state of a voting event and the amount and distribution of the votes, you would have to find all these transactions in the Tangle and count each vote. INX-Participation does that for you: it builds a database of all voting transactions for given events and provides REST API endpoints for clients to access this information. Node operators manually select which events they wish to track.
```

> There is no dedicated structure for a voting event on the Tangle. Instead, it is represented by multiple individual transactions with specific payloads. To access the state of a voting event and the amount and distribution of the votes, you would have to find all these transactions in the Tangle and count each vote. INX-Participation does that for you: it builds a database of all voting transactions for given events and provides REST API endpoints for clients to access this information. Node operators manually select which events they wish to track.

Reviewer 2: ```
There is no dedicated structure for a voting event on the Tangle. Instead, it is represented by multiple individual transactions with specific payloads. To access the state of a voting event and the amount and distribution of the votes, you would have to find all these transactions in the Tangle and count each vote.```
That sounds quite negative. Can't we change that part?

## Configuration

> The participation extension connects to the local Hornet instance by default.

Reviewer 2: We write HORNET with capital letters almost everywhere :P

Reviewer: There's actually a [fair mix](https://wiki.iota.org/search?q=hornet) going around. It'd be great if we standardized this.  Seeing that the repo uses HORNET we could go with that. Though, as it's not an acronym and most other projects don't capitalize all the letters it may be confusing or look out of place. 

Personally, I'd go with normal capitalization. That being said, either works as long as we're consistent. Wdyt?

> The participation extension connects to the running Hornet instance by default.

Reviewer: ```suggestion
The participation extension connects to a local Hornet on http://localhost:9892 instance by default.
```
or a local hornet instance.
